modules/vba_documents/lib/vba_documents/upload_error.rb

```diff
@@ -13,6 +13,7 @@ class UploadError < StandardError
     DOC105 = 'Invalid or unknown id'
     DOC106 = 'Maximum document size exceeded. Limit is 100MB per document'
     DOC107 = 'Empty payload'
+    DOC108 = 'Maximum page size exceeded'
```

Comment:
We should probably do like DOC106 and tell them what the size is in the error response.

Reply:
fair enough I'll add it there.